src/main/java/org/opensearch/security/dlic/rest/validation/AbstractConfigurationValidator.java

```diff
@@ -297,4 +316,25 @@ public String getMessage() {
     protected final boolean hasParams() {
         return param != null && param.length > 0;
     }
+
+    private boolean hasNullElement(JsonNode currentNode) {
+        if (currentNode.isArray()) {
+            List contentArray = DefaultObjectMapper.objectMapper.convertValue(currentNode, java.util.List.class);
+            if(contentArray.contains(null)){
+                return true;
+            }
+            ArrayNode arrayNode = (ArrayNode) currentNode;
+            Iterator<JsonNode> node = arrayNode.elements();
+            while (node.hasNext()) {
+                if( hasNullElement(node.next())) return true;
```

Comment:
nit: `if (hasNullElement`. While we don't enforce any particular code style, it will be better to make it consistent with the rest of the file. It is very uncommon to have space after `(` in the `if` condition.

Reply:
Done